stdlib/transactions/src/main/ballerina/transactions/commons.bal

```diff
@@ -33,13 +33,15 @@ map<TwoPhaseCommitTransaction> participatedTransactions = {};
 # This cache is used for caching HTTP connectors against the URL, since creating connectors is expensive.
 cache:Cache httpClientCache = new;
 
-final boolean scheduleInit = scheduleTimer(1000, 60000);
-
-function scheduleTimer(int delay, int interval) returns boolean {
-    (function() returns error?) onTriggerFunction = cleanupTransactions;
-    task:Timer timer = new(onTriggerFunction, (), interval, delay = delay);
-    timer.start();
-    return true;
+listener task:Listener timer = new({
+    interval: 5000,
```

Comment:
Initially the interval seems 60000 and any reason for changing this to 5000?

Reply:
Fixed.